helm/values.yaml

```diff
@@ -6,17 +6,16 @@ global:
   authentication:
     jwtPublicKeySecretName: map-colonies-jwks
 
-cloudProvider:
-  dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+cloudProvider: {}
 
 ### Useful for umbrella deployments in cases which no need geoserver serving ###
 enabled: true
 
 image:
   geoserverRepository: geoserver-os
-  geoserverTag: "v1.0.1-2.21.0-rc3"
+  geoserverTag: "latest"
```

Comment:
i don't think we should work with "latest" as it can possibly break our version with unwanted version, what do you think?